stock_picker.rb

```diff
@@ -0,0 +1,23 @@
+def stock_picker stock_prices
+  most_profitable_days = [0, 0]
+  max = 0
+  day = 0
+  stock_prices.each_with_index do |item, index|
+    day = index
+    stock_prices.each_with_index do |item_two, index_two|
+      day_two = index_two
+      if (day_two <= day)
+        next
+      end
+      if ((item_two - item) > max)
+        most_profitable_days = [day, day_two]
+        max = (item_two - item)
+      end
+
```

Comment:
Could use more descriptive variable names than "item" and "index". I used "buy_date" and "sell_date" and I think it's a lot more readable.